src/api/database/bulk-api.rst

```diff
@@ -418,24 +418,16 @@ in bulk must be handled separately; see :ref:`api/db/bulk_docs/validation`.
 Bulk Documents Transaction Semantics
 ====================================
 
-CouchDB supports two different modes for updating (or inserting)
-documents using the bulk documentation system. Each mode affects both
-the state of the documents in the event of system failure, and the level
-of conflict checking performed on each document. The two modes are:
-
--  **non-atomic**
-
-   The default mode is `non-atomic`, that is, CouchDB will only guarantee
-   that some of the documents will be saved when you send the request.
-   The response will contain the list of documents successfully inserted
-   or updated during the process. In the event of a crash, some of the
-   documents may have been successfully saved, and some will have been
-   lost.
+Bulk document operations are **non-atomic**. This means that CouchDB does not
+guarantee that documents included in a bulk update (or insert), will be saved
```

Comment:
Maybe `does not guarantee that all documents` ... `will be saved when the  response returns`. 

That is some documents might have been saved and some might have returned an error. As opposed to an atomic update were either all documents would be saved and then be immediately available in a read request, or if there is an error, none of them would be saved and would not appear in a subsequent read request.

Reply:
Your suggestion is closer to the original wording. The only reason I changed it is that it sounds as if it does guarantee that at least some documents will be saved, which of course it cannot guarantee, either.

What about:

`This means that CouchDB does not guarantee that any individual document included in the bulk update (or insert) will be saved.`

?

Reply:
That's sounds good. I was thinking how to highlight that `non-atomic` means there are no guarantees regarding the group of documents being updated together.

Say if the user has two documents: <doc1> and <doc2> and they have to somehow be kept in sync (that is updated together). They might think `_bulk_docs` with `[<doc1>, <doc2>]` would do that, but it turns out <doc1> might be updated and a client would read the database at just the right time to only see the update to <doc1> but not <doc2>.

Otherwise, any individual document has the same guarantees as the if the document would be updated via the <db>/<doc_id> endpoint.

Reply:
I agree, that's a fair concern, and my original wording didn't make that point very well.

I've updated the PR to include my last suggested wording.